src/IssueViz.UnitTests/Models/AnalysisIssueVisualizationTests.cs

```diff
@@ -194,7 +194,9 @@ public void SetSpan_HasSubscribers_NotifiesSubscribers()
         public void SetIsSuppressed_HasSubscribers_VerifyRaised()
         {
             var testSubject = CreateTestSubject();
-            
+
+            Assert.IsFalse(testSubject.IsSuppressed);
```

Comment:
Minor: `testSubject.IsSuppressed.Should().BeFalse();` would be better (it's cleaner not to mix old-style `Assert.IsTrue/False` with `FluentAssertions`).